File: src/main/java/com/bombproj/model/DataTableIndex.java
package com.bombproj.model;

import lombok.Data;

import java.util.Date;

@Data
public class DataTableIndex {

    private Long id;
    private String indexName;
    private String fieldNames;
    private String indexType;
    private String indexMethod;
    private String projectId;
    private Long databaseId;
    private String datatableId;
    private Integer state;
    private String version;
    private String userId;
    private Date createTime;
    private Date updateTime;
    private Integer sequence;

}

File: src/main/scala/com/bombproj/model/DataTableIndex.scala
package com.bombproj.model

import java.util.Date

import scala.beans.BeanProperty

case class DataTableIndex() {

    @BeanProperty var id: Long = _
    @BeanProperty var indexName: String = _
    @BeanProperty var fieldNames: String = _
    @BeanProperty var indexType: String = _
    @BeanProperty var indexMethod: String = _
    @BeanProperty var projectId: String = _
    @BeanProperty var databaseId: Long = _
    @BeanProperty var datatableId: String = _
    @BeanProperty var state: Int = _
    @BeanProperty var version: String = _
    @BeanProperty var userId: String = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var updateTime: Date = _
    @BeanProperty var sequence: Int = _

}
